KarhooUISDK.xcodeproj/project.pbxproj

```diff
@@ -4574,6 +4599,7 @@
 				407EC7B022CF564E00F24CFC /* KarhooGradients.swift in Sources */,
 				407EC7CC22CF564E00F24CFC /* KarhooAddressBarFieldView.swift in Sources */,
 				6BB940542754D47800478ABD /* LoyaltyViewModel.swift in Sources */,
+				7B7404452975A9D5000ED60A /* KarhooNewTextField.swift in Sources */,
```

Comment:
Should we rename this to `KarhooMaterialDesignTextField`? It will be a while until we remove the old `KarhooTextField` and we shouldn't be left with "new" in the name until then.